birdhouse/components/thredds/default.env

```diff
@@ -8,8 +8,8 @@ export THREDDS_ORGANIZATION="Birdhouse"
 export THREDDS_ADDITIONAL_CATALOG=""
 
 # All default values for data locations are set to ensure backward compatibility
-export THREDDS_DATASET_LOCATION_ON_CONTAINER='/pavics-ncml'  # this default is for backward compatibility
-export THREDDS_SERVICE_DATA_LOCATION_ON_CONTAINER='/pavics-data'  # this default is for backward compatibility
+export THREDDS_DATASET_LOCATION_ON_CONTAINER='/birdhouse-ncml'  # this default is for backward compatibility
+export THREDDS_SERVICE_DATA_LOCATION_ON_CONTAINER='/birdhouse-data'  # this default is for backward compatibility
```

Comment:
No, this is not backward compat since all the `.ncml` will have already been using `/pavics-data` to access all the `.nc` files it is aggregating.

A sample of those `.ncml`: https://github.com/Ouranosinc/pavics-vdb/blob/bebd3f3dc597f503ce2bfa82dec9d7be565f2fd9/1-Datasets/simulations/climex/day_climex-crcm5_historical%2Brcp85.ncml#L37 that would break on our end.

That said, if need be, I can override that variable to keep the old `/pavics-data` so we do not have to mass update all our `.ncml` files.

Reply:
@tlvu yeah that makes sense. I've addressed that in migration guide here:

https://github.com/bird-house/birdhouse-deploy/blob/92dd7f6ee32958dcf9da86fe37343e72c5afc88c/CHANGES.md?plain=1#L59-L68

So you should override these two variables so that the point to the old default for you.